packages/core/echo/echo-db/src/query/filter.ts

```diff
@@ -230,12 +230,10 @@ const filterMatchInner = (filter: Filter, core: AutomergeObjectCore): boolean =>
         return false;
       }
     } else {
-      if (!type) {
+      if (!type && filter.type.itemId !== 'Expando') {
```

Comment:
extract 'Expando' to a constant.
is there any way for this and Expando deifnition in echo-schema to use the same constant?